jsonnet/kube-prometheus/kube-state-metrics/kube-state-metrics.libsonnet

```
@@ -34,6 +34,8 @@
                               spec+: {
                                 containers: std.map(function(c) c {
                                   ports: null,
+                                  livenessProbe: null,
+                                  readinessProbe: null,
```

```suggestion
                                  livenessProbe:: null,
                                  readinessProbe:: null,
```
Would actually hide them.